modules/rostests/apitests/ws2_32/WSAAsync.c

```diff
@@ -119,6 +119,7 @@ START_TEST(WSAAsync)
 
```

Comment:
FYI I have a fix for this one in the queue.

Reply:
Remember to revert this change, when you fixed it properly 😛